apistar/commands/create_tables.py

```diff
@@ -41,9 +61,51 @@ def django_showmigrations() -> None:
     """
     Show Migrations DjangoBackend.
     """
-    from apistar.cli import get_current_app
-    from apistar.backends.django_backend import DjangoBackend
-    app = get_current_app()
-    db_backend = DjangoBackend.build(settings=app.settings)
+    db_backend = _get_django_backend()
     db_backend.showmigrations()
     click.echo("showmigrations")
+
+
+def alembic_init() -> None:
+    """
+    Alembic Initialize SQLAlchemyBackend.
+    """
+    db_backend = _get_sqlalchemy_backend()
+    db_backend.initialize()
+    click.echo("Initialize alembic migrations")
+
+
+def alembic_revision(message: Message) -> None:
+    """
+    Alembic Create Revision SQLAlchemyBackend.
+    """
+    db_backend = _get_sqlalchemy_backend()
+    db_backend.revision(message=message)
+    click.echo("revision: %s" % message)
+
+
+def alembic_downgrade(revision: Revision) -> None:
+    """
+    Alembic Downgrade SQLAlchemyBackend.
+    """
+    db_backend = _get_sqlalchemy_backend()
+    db_backend.downgrade(revision=revision)
+    click.echo("Downgrade to revision: %s" % revision)
+
+
+def alembic_upgrade(revision: Revision) -> None:
+    """
+    Alembic Upgrade SQLAlchemyBackend.
+    """
+    db_backend = _get_sqlalchemy_backend()
+    db_backend.upgrade(revision=revision)
+    click.echo("upgrade to revision: %s" % revision)
```

Comment:
Injecting the `Console` component and calling `console.echo(...)` is now the right approach here.